src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -22,6 +22,30 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>CardExpirationDate__c</fullName>
+        <description>The expiration month and year of the card used for this Recurring Donation.</description>
```

Comment:
Should the Help for these two fields use "credit card" instead of just "card"?
@sam-knox ?

@RadSFDeveloper If the field only has MM/YY or MM/YYYY should the field length be either 5 or 7?

Reply:
It is `MM/YYYY `. I can change it to 7 to be exact.

Reply:
Out of curiosity, do you think we should parse this into two fields:
cardExpirationMonth__c (2) and cardExpirationYear__c (4)

Reply:
@force2b - Let me ask if there's a current convention. I was going to say use "credit card" but some people use debit cards.

Reply:
Yeah, let's go with just "card" @force2b